src/scrapers/mizrahi.ts

```diff
@@ -154,7 +157,30 @@ class MizrahiScraper extends BaseScraperWithBrowser {
   }
 
   async fetchData() {
+    const numOfAccounts = (await this.page.$$(accountDropDownItemSelector)).length;
+    try {
+      const results: TransactionsAccount[] = [];
+      for (let i = 0; i < numOfAccounts; i += 1) {
+        await this.page.$$eval(accountDropDownItemSelector, (els, i) => (els[i] as HTMLElement).click(), i);
+        results.push(await this.fetchAccount());
+      }
+
+      return {
+        success: true,
+        accounts: results,
+      };
+    } catch (e) {
+      return {
+        success: false,
+        errorType: ScraperErrorTypes.Generic,
+        errorMessage: (e as Error).message,
+      };
+    }
```

Comment:
Note it means "everything or nothing".
Do you want me to `try-catch` each account instead?

For example, I needed to fix the pending transactions that didn't work, because it blocked the next account.

Reply:
👍  I believe it should be all or nothing because otherwise the user might get partial information without any notice about it